service-node/backend/src/main/scala/org/enterprisedlt/fabric/service/node/ServiceNode.scala

```diff
@@ -135,6 +139,21 @@ object ServiceNode extends App {
             )
         )
 
+        val fileEnpointContext = new ContextHandler("/")
+        val tmpLocation: String = "/tmp/files/" // the directory location where files will be stored
+        val maxFileSize: Int = 100 * 1024 * 1024 // the maximum size allowed for uploaded files
+        val maxRequestSize: Int = 100 * 1024 * 1024 // the maximum size allowed for multipart/form-data requests
+        val fileSizeThreshold: Int = 1024 // the size threshold after which files will be written to disk
+        val multipartConfig: MultipartConfigElement = new MultipartConfigElement(tmpLocation, maxFileSize, maxRequestSize, fileSizeThreshold);
```

Comment:
put the vars above to `UploadHandler` contructor args, initialize `multipartConfig` in `UploadHandler`

Reply:
done.